### PR TITLE
Fix 100 passive impling task

### PR DESCRIPTION
**Pull request by nwjgit**
### Description:
- Fix master league task 4151 "'Catch 100 of every impling passively (excluding Lucky implings)"
Discord: https://discord.com/channels/342983479501389826/342983479501389826/1103108637511143484

### Changes:
- Fix task to only check for 50 mystery imp

### Other checks:
- [X] I have tested all my changes thoroughly.
Discord testing: https://discord.com/channels/940758552425955348/1162793245194072074/1172416337243545632

### CODE REVIEW

File: src/lib/leagues/masterTasks.ts [hunk 1]
```diff
@@ -1090,12 +1092,16 @@ export const masterTasks: Task[] = [
 		}
 	},
 	{
-		id: 4151,
+		id: 4151, // 50 mystery impling, 100 of every other imping (excluding Lucky implings)
 		name: 'Catch 100 of every impling passively (excluding Lucky implings)',
 		has: async ({ userStats }) => {
```
Comment (gc) [inline src/lib/leagues/masterTasks.ts:1097]: If we're adding this extra special case with mystery implings, the title should mention it, not in a comment users cant see.

Comment (nwjgit) [reply src/lib/leagues/masterTasks.ts]: Changed it to "Catch 50 mystery impling and 100 of every other imping passively (excluding Lucky implings)" let me know how that sounds

**Comment by gc** [comment]
You committed the extra .txt file which isnt meant to be committed. I know I need to make it stop generating that.

**Comment by nwjgit** [comment]
Fixed sorry about that

**Pull request merged**